pkg/operator/target_config_reconciler.go

```diff
@@ -412,61 +412,61 @@ func (c *TargetConfigReconciler) manageConfigMap(descheduler *deschedulerv1.Kube
 			if strategy.Params == nil {
 				strategy.Params = &deschedulerapi.StrategyParameters{}
 			}
-			// skip strategies which don't support namespace exclusion yet
-			if name == "LowNodeUtilization" {
-				if descheduler.Spec.ProfileCustomizations != nil && descheduler.Spec.ProfileCustomizations.DevLowNodeUtilizationThresholds != nil {
-					if strategy.Params == nil {
-						strategy.Params = &deschedulerapi.StrategyParameters{}
+			if descheduler.Spec.ProfileCustomizations != nil && descheduler.Spec.ProfileCustomizations.DevLowNodeUtilizationThresholds != nil {
```

Comment:
Let's keep `name == "LowNodeUtilization"` as well so `strategy.Params.NodeResourceUtilizationThresholds` is set only when `name == "LowNodeUtilization"` to avoid updating `strategy.Params.NodeResourceUtilizationThresholds` if not needed.